css/grayscale.css

```diff
@@ -381,3 +381,82 @@ img::-moz-selection {
 body {
     webkit-tap-highlight-color: rgba(255,255,255,.2);
 }
+
+
+
+/****************************
+* Social Share Buttons
+****************************/
+.share-buttons {
```

Comment:
Do we really need a new class for this? Use bootstrap's buttons. Also, look into fontawesome for logos of websites.